skbio/tree/tests/test_tree.py

```diff
@@ -633,6 +633,18 @@ def test_set_max_distance(self):
         self.assertEqual(tip_a[0] + tip_b[0], 1.6)
         self.assertEqual(sorted([tip_a[1].name, tip_b[1].name]), ['b', 'e'])
 
+    def test_set_max_distance_tie_bug(self):
```

Comment:
I think we should also have a test that midpoint rooting now works when there are equal branch lengths. I started a test of this in a PR that I had:

```
    def test_root_at_midpoint_equal_lengths(self):
        nwk = u"(((a:1,b:1)c:2),((d:3,e:4)f:5))root;"
        t = TreeNode.read(StringIO(nwk))
        obs = t.root_at_midpoint()
        exp_nwk = u""
        self.assertEqual(str(obs), exp_nwk)
```